app/assets/javascripts/angular/angular-config.js.erb

```diff
@@ -3,25 +3,27 @@ var app = angular.module('fritter', ['ngRoute']);
 app.config(['$routeProvider',function($routeProvider){
   $routeProvider.when('/', {
     templateUrl: "<%= asset_path('pin-index.html') %>",
-    controller: 'HomeCtrl',
+    controller: 'HomeCtrl'
   })
 }])
 
 app.config(['$routeProvider',function($routeProvider){
```

Comment:
There's no need to repeat app.config all the time, neither $routeProvider, for each route. you can do something like:
```
app.config(['$routeProvider',function($routeProvider){
    $routeProvider
        .when('...', {   ...   })
        .when('...', {   ...   })
        .when('...', {   ...   })
        .when('...', {   ...   })
```